doc/commands.md

```diff
@@ -0,0 +1,41 @@
+# CLI commands documentation
+
+For each function the get_parser method is the same:
+* The function can add a node, network, or port, depending on the class. 
+* The parser is then returned. 
+
+## Node/Network
+
+Links to the commands are here:
+* [Attach](#attach)
+* [Detach](#detach)
+* [List](#list)
+
+
+### <a name="attach"></a>Attach
+
+The attach command will attack a network to a node. 
+
+* `openstack esi node network attach <node> --port <port> --network <network>` 
+    * Exactly one port or network is needed.
+    * port: name or UUID of the port.
+    * node: name or UUID of the node.
```

Comment:
Can you move `node` above `port`? This way the order of parameters matches.

Reply:
sure!